### PR TITLE
Assignment1 - Ethan Guerin

### PR DESCRIPTION
**Pull request by EthanAGi**

### CODE REVIEW

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public static void Main(string[] args)
 
     public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
     {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
+        return ((double)numberCorrectAnswers / numberOfQuestions * 100) + "%";
```
Comment (MattA41) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:24]: why did you use a double over a float?

**Review by EthanAGi — COMMENTED** [review]
Good idea! I'll make that change!

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs
Comment (BenjaminMichaelis) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:1]: This program when run has a runtime error

Comment (EthanAGi) [reply PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:1]: I believe I fixed it!

**Pull request closed without merging**